### ISSUE TITLE
Bug: Have to type `{peer_id}.awl` into browser url bar before anything else on system connects to peer using that address.

### ISSUE THREAD

**Issue by dillfrescott** [anywherelan/awl#79]
Experienced on Windows 11 Pro.

Very odd bug indeed. Happens after every restart of awl service.

**Comment by pymq** [comment]
@dillfrescott I tried to reproduce this on Windows 11 Home 22H2 build 22621.1105

1. Added peer
2. Close browser
3. Close awl, start again
4. Ping/curl `{peer_id}.awl` - error, couldn't resolve host
5. Open this address in browser - works
6. Retry 4. - it works

Investigated, part of the problem is that awldns "announce" lowercase `peer_id` for host.

So, as a temporal workaround, if you use `{peer_id}.awl` in lowercase, it's working in 4. in the steps above
